### ISSUE TITLE
Improve accuracy above 0.98398 on simplified MNIST classification

### ISSUE THREAD

**Issue by dgonzo** [ziff/internship2017#2]
@ziff/owners created [Challenge Task 1](https://github.com/ziff/internship2017/blob/master/Challenges/Task_1_Basic_CONVNET_w_Simplified_MNIST.ipynb)

# ZIFF Summer Internship 2017 Challenge Task 1: Improve accuracy with GridSearch

## Objective

Demonstrate an understanding of hyperperameter optimization using sklearn GridSearch on a convolutional deep net against a simplified MNIST digit regcognition by improving out-of-sample accuracy above 0.98398.

## Submission Criteria

  * Follow the instructions in Concepts > Challenge Submission in this project
  * Your solution must submitted by Friday June 9th 12:00 midnight

## Resources

  * [How to Grid Search Hyperparameters for Deep Learning Models in Python With Keras](http://machinelearningmastery.com/grid-search-hyperparameters-deep-learning-models-python-keras/)
  * [Deep Learning for Developers](https://www.youtube.com/watch?v=lr3ZZAyHgsM)
  * [Lessons learned from 100 deep learning models](https://www.youtube.com/watch?v=HZfNlzziICQ)
  * [Machine Learning Mastery](http://machinelearningmastery.com/)

**Comment by bentaylordata** [comment]
No GPU resources should be needed because the problem and the next depth are relatively simple. Main think here that benefits everyone is becoming familiar with the parameters and understanding how to wrap a deep net into a class for sklearn consumption. The blog post above should be a great template, but total flexibility here as well. It is more important to engage and learn something with the task
